lib/kafka_ex/protocol/produce.ex

```diff
@@ -44,9 +47,10 @@ defmodule KafkaEx.Protocol.Produce do
     - key: is used for partition assignment, can be nil, when none is provided
     it is defaulted to nil
     - value: is the message to be written to kafka logs.
+    - timestamp: timestamp (`kafka_version: "kayrock"` ONLY)
     """
-    defstruct key: nil, value: nil
-    @type t :: %Message{key: binary, value: binary}
+    defstruct key: nil, value: nil, timestamp: nil
+    @type t :: %Message{key: binary, value: binary, timestamp: integer}
```

Comment:
shouldn't the [timestamp](https://github.com/kafkaex/kafka_ex/issues/261#issuecomment-530973125) be optional?
```suggestion
    @type t :: %Message{key: binary, value: binary, timestamp: integer | nil}
```